### ISSUE TITLE
Storefront Node Switch

### ISSUE THREAD

**Issue by Volas171** [Jexactyl/Jexactyl#71]
### Is there an existing feature request for this?

- [X] I have searched the existing issues before opening this feature request.

### Describe the feature you would like to see.

As the storefront currently is, it's fine, however, all servers get created to a specific node, my idea is to implement an option to let users select the node they want (i also suggest some control on the admin UI that allows restricting what nodes users can create from, however, that's not as important) so transfers to other nodes don't have to be done manually.

### Describe the solution you'd like.

basically a new dropdown on the storefront that lets users select between nodes when creating servers and if possible the ability to limit what nodes they can create servers from on the admin UI.
